### PR TITLE
Support ordering belongs to fields

### PR DESCRIPTION
**Pull request by pedro**
Hey folks!

Do I need to create a new class and stub the hell out of it to test this? I'd much rather have a sample AR model available to tests, but couldn't find any in the suite. Thoughts?

See #278

### CODE REVIEW

**Comment by pedro** [comment]
Rebased!

File: lib/administrate/field/belongs_to.rb [hunk 1]
```diff
@@ -24,7 +24,11 @@ def selected_option
       private
 
       def candidate_resources
-        associated_class.all
+        scope = associated_class
+        if options[:order]
+          scope = scope.order(options[:order])
+        end
+        scope.all
```
Comment (BenMorganIO) [inline lib/administrate/field/belongs_to.rb]: Refactor suggestion:

```ruby
order = options.delete(:order)
order ? associated_class.order(order) : associated_class.all
```

**Comment by BenMorganIO** [comment]
@pedro Tests would be great. If you need a sample AR model to get your tests working, that should be no problem :). Or you could just take one of the existing models and add new "magic" to them.

**Comment by carlosramireziii** [comment]
@pedro I know it's been a while... Are you still interested in getting this PR merged? I think we'd need to get some tests in here and maybe some documentation on the new feature. 

LMK what you think.

**Comment by nickcharlton** [comment]
Closing this as #939 has since been merged.

**Pull request closed without merging**